build-system/npm-publish/utils.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Get extensions to be published on npm
+ * @return {Array<any>}
+ */
+function getExtensions() {
+  const bundles = require('../compile/bundles.config.extensions.json');
+  const extensions = bundles
+    .filter((bundle) => bundle.options?.npm)
+    .map((bundle) => ({
+      'extension': bundle.name,
+      'version': bundle.version,
+    }));
+  return extensions;
+}
+
+/**
+ * Get semver from extension version and amp version
```

Comment:
Maybe link to `getVersion()` in `build-system/compile/internal-version.js` as a way to document / provide rationale for the arguments to `split()` / `slice()` in this function?